### ISSUE TITLE
Some minor issues

### ISSUE THREAD

**Issue by dc42** [dxoverdy/Integrating-Alpha2MQTT-with-Home-Assistant#1]
Thanks so much for providing these instructions! I am new to Home Assistant and I would have had a steep learning curve without them.

I encountered a few things that didn't quite match your documentation, likely because Home Assistant and its addons have changed:

- Entering my DuckDNS URL and token was straightforward without resorting to YAML. I guess they have fixed their configuration page.
- Node RED wouldn't start. The log messages indicated two problems: lack of a SSL certificate, and un-configured credential-secret. So in the Node RED configuration I disabled SSL and entered a string for credential-secret.
- After starting Node RED, the first script import worked. For each subsequent import there was a popup "Some of the nodes you are importing already exist in your workspace" with options "Cancel", "View nodes" and "Import copy". I didn't know whether to click "Import copy" or to click "View nodes" and if so what to do afterwards. In the end I selected "View Nodes" and then "Replace", but each time I had to correct the broker IP address afterwards.
- When adding dashboard Alpha there is no mdi:battery icon, instead there are several mdi:battery-xxx icons for varous xxx
- When adding the dashboard, the "'Custom element doesn't exist: apexcharts-card' error is still present. But re-downloading it didn't fix it. [EDIT: restarting Home Assistant after adding the dashboard and then going into HACS and re-downloading the addons did fix it.]
- When adding dashboard Alpha Control there is no mdi:remote icon. I chose mdi:remote-desktop.

**Comment by dxoverdy** [comment]
No worries at all.  I'll be honest, I'm doing a lot of this blindfold.  I
don't have an PI with home assistant.  I'm doing it based on some historic
knowledge and a virtual machine.
I'm sure the instructions make reference to this by way of 'It may prompt
"Some of the nodes you are importing already exist in your workspace", just
choose 'Import Copy''
Perhaps it isn't obvious enough.

Glad you're moving it to the next stage.  How has the device been
performing since you brought it to life.  Reliable?

Dan

**Comment by dc42** [comment]
I restarted HA, went into HACS and this time it did show 5 Lovelace elements not loaded (previously it didn't). I re-downloaded them all and the power chart is now working.

There are quite a few "Unknown" elements showing in the Alpha dashboard, but I guess some of them are only updated by the once-a-day flow, and others may not be relevant because I haven't configured the corresponding features e.g. forced discharge. I didn't import the NodeRED TOU flow because I am not yet on a TOU tariff.

My main motivation for doing this is that my 2-year electricity fix at a good rate expires in September and if I can better control the inverter I will switch to a TOU tariff. Ideally I will switch to Octopus Agile at least during the summer. I've already worked out how to get remote control of my Hypervolt EV charger, see https://miscsolutions.wordpress.com/2022/10/09/home-energy-management-system-part-4-characterising-the-hypervolt-current-transformer/, and my iBoost+ solar hot water diverter, see https://miscsolutions.wordpress.com/2022/10/02/home-energy-management-system-part-2-hacking-the-iboost-protocol/.

Thanks again for all your hard work on this! I'll let you know how reliable the unit is in a few days. I'm considering adding counts of CRC and timeout errors to the data sent to MQTT and/or displayed on the OLED, for diagnostic purposes.

Btw I'm running HA on a Pi 3B+. For Alpha2MQTT I'm using the blue MAX485 board powered from +5V, with a voltage divider to reduce the receive output signal from 5V to less than 3.3V. My next step is to design a 3D printable enclosure to house the 4 modules.
